generators/idea-generator/tests/org/jetbrains/kotlin/generators/tests/idea/GenerateTests.kt

```diff
@@ -1147,6 +1147,10 @@ fun main(args: Array<String>) {
                 model("checker/diagnosticsMessage", excludedPattern = excludedFirTestdataPattern)
             }
 
+            testClass<AbstractHighLevelIntentionTest> {
+                model("intentions/importAllMembers", pattern = "^([\\w\\-_]+)\\.(kt|kts)$")
+                model("intentions/importMember", pattern = "^([\\w\\-_]+)\\.(kt|kts)$")
```

Comment:
Please, extract this regexp into pattern, so we can easily reuse it for other intentions